sematic/api/server.py

```diff
@@ -183,27 +189,27 @@ def parse_arguments() -> argparse.Namespace:
     return parser.parse_args()
 
 
-def run_wsgi(daemon: bool):
-    options = {
-        "bind": f"{get_config().server_address}:{get_config().port}",
-        "workers": get_config().wsgi_workers_count,
-        "worker_class": "geventwebsocket.gunicorn.workers.GeventWebSocketWorker",
-        "daemon": daemon,
-        "pidfile": get_config().server_pid_file_path,
-        "logconfig_dict": make_log_config(log_to_disk=True),
-        "certfile": os.environ.get("CERTIFICATE"),
-        "keyfile": os.environ.get("PRIVATE_KEY"),
-    }
+def app():
+    dictConfig(make_log_config(log_to_disk=True, level=logging.DEBUG))
     register_signal_handlers()
-    SematicWSGI(sematic_api, options).run()
+
+    if os.environ.get("SEMATIC_SOCKET_IO_ONLY", "") != "":
+        sio = init_socket_io_server(make_async=True)
+        register_sio_server(sio)
+        full_app = socketio.ASGIApp(sio, starlette_app)
+    else:
+        full_app = WsgiToAsgi(sematic_api)
+
+    return full_app
 
 
 if __name__ == "__main__":
     args = parse_arguments()
     switch_env(args.env)
 
     if args.debug:
-        run_socketio(args.debug)
+        run_locally(args.debug, make_daemon=False)
 
     else:
-        run_wsgi(False)
+        print("To run the server in production, please launch it using uvicorn directly")
```

Comment:
Added a reference to the call in the Dockerfile, as it will be more comprehensive and up-to-date than what we'd want to put directly in the console output.